enos/modules/map2list/main.tf

```diff
@@ -0,0 +1,5 @@
+variable "map" {}
```

Comment:
context: we can't currently use built in functions (like `keys()`) in enos scenarios themselves, this is on the roadmap though